### DISCUSSION TITLE
Alt (option) key in MacOS isn't working as expected.

### DISCUSSION THREAD

**Discussion by amjith** [ghostty-org/ghostty#2845] [Q&A] [answered]
I use tmux and `Alt + i` is my prefix key. But when I press `Alt + i + v` (vertical split in my tmux conf) the character `^v` is inserted. So Tmux never receives the key stroke to invoke the vertical split. 

I tried disabling all keybinds in the config using `keybind = clear`. That disabled the default ghostty keybindings but the `Alt + i` is still not being passed on to tmux.

I checked the readline [keybindings](https://www.bigsmoke.us/readline/shortcuts) to see if this is something to override in readline but I didn't see it listed there either. 

Is there a way to disable the current `Alt + i` behavior so tmux can continue using that as a bind key?

**Comment by mitchellh** [comment]
Converted to a discussion until the research can be done to definitively say this is a bug. Open questions:

1. What is the intended key encoding of `alt + i + v`?
2. I don't use tmux, can you provide a step by step reproduction from a non-tmux user (assume tmux is installed) to reproduce this?
3. What operating system is this on? Keyboard input varies dramatically between Linux and macOS.

**Reply by amjith** [reply, marked as answer]
Resolved!

I am on MacOS. The option key wasn't sending Alt. I found out there is a config called `macos-option-as-alt = true` and adding that to my `~/.config/ghostty/config` file fixed the issue.
